collectors/charts.d.plugin/ap/metadata.yaml

```
@@ -3,52 +3,86 @@ meta:
   module_name: ap
   monitored_instance:
     name: Access Points
-    link: ''
+    link: "https://learn.netdata.cloud/docs/data-collection/networking-stack-and-network-interfaces/linux-access-points"
     categories:
-    - data-collection.networking-stack-and-network-interfaces
-    icon_filename: 'netdata.png'
+      - data-collection.networking-stack-and-network-interfaces
+    icon_filename: "netdata.png"
```

figured there is a nicer icon to use here

```suggestion
    icon_filename: "network-wired.svg"
```